schema.graphql

```diff
@@ -70,6 +70,7 @@ type Dataset {
   status: JobStatus
   inputPath: String
   uploadDateTime: String
+  fdrCounts: String
```

Comment:
I'd propose introducing a small data type (with fields `fdrLevel` & `annotationCount`) for the return value here and letting GraphQL do all format handling on its own instead of manually doing `stringify` and `parse`